core/domain/suggestion_registry_test.py

```diff
@@ -3845,3 +3845,390 @@ def test_create_contribution_stats_summary(self) -> None:
         self.assertDictEqual(
             contribution_summary.to_dict(), expected_contribution_summary
         )
+
+
+class TranslationSubmitterTotalContributionStatsUnitTests(
+    test_utils.GenericTestBase):
+    """Tests for the TranslationSubmitterTotalContributionStats class."""
+
+    SUGGESTION_LANGUAGE_CODE: Final = 'es'
+    USER_ID_1: Final = 'uid_01234567890123456789012345678912'
+    TOPIC_IDS_WITH_TRANSLATION_SUBMISSIONS: Final = ['topic1', 'topic2']
+    RECENT_REVIEW_OUTCOMES: Final = ['accepted', 'rejected']
+    RECENT_PERFORMANCE: Final = 2
+    OVERALL_ACCURACY: Final = 2.0
+    SUBMITTED_TRANSLATIONS_COUNT: Final = 2
+    SUBMITTED_TRANSLATION_WORD_COUNT: Final = 100
+    ACCEPTED_TRANSLATIONS_COUNT: Final = 1
+    ACCEPTED_TRANSLATIONS_WITHOUT_REVIEWER_EDITS_COUNT: Final = 0
+    ACCEPTED_TRANSLATION_WORD_COUNT: Final = 50
+    REJECTED_TRANSLATIONS_COUNT: Final = 0
+    REJECTED_TRANSLATION_WORD_COUNT: Final = 0
+    FIRST_CONTRIBUTION_DATE = datetime.date.today()
+    LAST_CONTRIBUTION_DATE = (
+        datetime.date.today() - datetime.timedelta(25))
+
+    def test_create_translation_submitter_total_stats(self) -> None:
```

Comment:
It looks like the `to_dict` method is the method under test here, so let's include that in the test name. Ditto below.

```suggestion
    def test_to_dict_returns_dict(self) -> None:
```

Reply:
Done.